packages/prop-types/src/array-of-max-length.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types'
+
+import PropTypeError from './prop-type-error'
+import { createChainableTypeChecker, getCircularReplacer } from './utils'
```

Comment:
add file ext. here and a few other places in the pr.   ...not sure if I should mark them all.  is it noise or helpful.  What do you think?

Reply:
i'll go through and search/destroy

Reply:
i need to see if we can update eslint to warn me about this...cause i never remember